src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -561,15 +577,15 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
 
       contextDispatch({ type: ThorchainSaversDepositActionType.SET_LOADING, payload: true })
 
-      const { getIsTradingActive } = swapperApi.endpoints
-      const { data: isTradingActive } = await appDispatch(
-        getIsTradingActive.initiate({
-          assetId,
-          swapperName: SwapperName.Thorchain,
-        }),
-      )
+      // Was the pool active when it was fetched at the time of the component mount
+      if (isTradingActive === false) {
+        throw new Error(`THORChain pool halted for assetId: ${assetId}`)
+      }
```

Comment:
this case might not be required because you refetch it anyway

Reply:
For completeness and perf - if it's halted at this point, it means it just became halted between overview and confirm mount, so it pretty much is guaranteed to still be halted and doesn't need a refetchy